requirements.txt

```diff
@@ -341,8 +345,8 @@ six==1.15.0 \
     --hash=sha256:30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259 \
     --hash=sha256:8b74bedcbbbaca38ff6d7491d76f2b06b3592611af620f8426e82dddb04a5ced \
     # via gcloud, google-api-core, google-auth, google-cloud-bigquery, google-resumable-media, jsonschema, oauth2client, packaging, protobuf, pyrsistent, pytest-xdist, python-dateutil
-smart_open==1.10.0 \
-    --hash=sha256:bea5624c0c2e49987c227bdf3596573157eccd96fd1d53198856c8d53948fa2c \
+smart_open==2.1.1 \
+    --hash=sha256:51b05acd85ec007e1d4dcdbf2bbf917218a45026f37d559559401114bb5e5840 \
```

Comment:
@scholtzan said in a previous PR that this may break public json uploads https://github.com/mozilla/bigquery-etl/pull/893#discussion_r433423416

Reply:
Tested it with this smart_open version and it worked.